tests/tests/fv3_gfs_v16_RRTMGP

```diff
@@ -75,10 +87,14 @@ export LIST_FILES="atmos_4xdaily.tile1.nc \
 export_fv3
 
 DT_ATMOS="1200"
+export FHMAX=48
+export RESTART_INTERVAL=24
+
+export NSTF_NAME=2,1,0,0,0
```

Comment:
Is this NSTF change required for the RRTMGP test? this is different from gfs_v16 configuration, which uses 2,0,0,0,0.

Reply:
I did this to be consistent with the fv3_gfs_v16 RT.